### PR TITLE
IDE-603 Plugin + sibling folders

### PR DESCRIPTION
**Pull request by dehilsterlexis**
Fixed IDE-603

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith can't seem to get the extra commit to disappear. Need some git help.

**Comment by richardkchapman** [comment]
try

git rebase --onto master HEAD^ && git push origin --force

File: comms/AttributeImpl.h [hunk 1]
```diff
@@ -19,34 +19,48 @@ class CAttributeBase : public clib::CLockableUnknown
     virtual const TCHAR *GetText(bool refresh, bool noBroadcast = false) const = 0;
     virtual const TCHAR *GetLabel() const = 0;
 
-    bool locatePlugin(const std::string & batchFile, boost::filesystem::path & foundFolder) const
+    bool ParentFolderSearch(const std::string & batchFile, boost::filesystem::path & foundFolder, int level, bool pluginFolder) const
```
Comment (GordonSmith) [inline comms/AttributeImpl.h]: Minor:  Should be camel case - how about "pluginFolderExists"?

**Comment by dehilsterlexis** [comment]
@GordonSmith back to you

**Comment by GordonSmith** [comment]
Looks good.

**Pull request merged**